### ISSUE TITLE
Top crash in customer:money

### ISSUE THREAD

**Issue by mehmetf** [flutter/flutter#69683]
The following is the top crash in customer:money iOS release.

```
Thread 13  (id: 0x00005207) CRASHED [EXC_BAD_ACCESS / KERN_INVALID_ADDRESS @ 0x00000100 ]Show exception record
0x00000001081e127c(Flutter + 0x003e527c)
0x00000001081e1264(Flutter + 0x003e5264)
0x00000001082d76b4(Flutter + 0x004db6b4)
0x00000001082d5dec(Flutter + 0x004d9dec)
0x00000001082d59d0(Flutter + 0x004d99d0)
0x0000000108357f5c(Flutter + 0x0055bf5c)
0x00000001083104e8(Flutter + 0x005144e8)
0x00000001f2a91b6c(libsystem_pthread.dylib + 0x00009b6c)_pthread_start
```

User reports are all we have. This crash corresponds to 12% of all their crashes making it their #1 crash.

**Comment by mehmetf** [comment]
@flutter-symbolizer-bot https://github.com/flutter/flutter/issues/69683#issue-735483526 internal engine#2eac514f26a676d1df0821e815d501149d3ead7f ios release arm64

**Comment by mehmetf** [comment]
@mraleph This is a VM issue.

**Comment by aam** [comment]
I believe this is a duplicate of https://github.com/flutter/flutter/issues/68603 the fix for which landed late last week.

**Comment by aam** [comment]
Closing as duplicate

**Comment by ghostgzt** [comment]
> I believe this is a duplicate of #68603 the fix for which landed late last week.

Crash many times!

Application received signal SIGSEGV (null)

**Comment by mehmetf** [comment]
A similar crash:

```
Thread 13  (id: 0x00005477) CRASHED [EXC_BAD_ACCESS / KERN_INVALID_ADDRESS @ 0x175a3c3a ]Show exception record
0x000000010611a48c(Flutter + 0x003e648c)
0x000000010611a468(Flutter + 0x003e6468)
0x0000000106211948(Flutter + 0x004dd948)
0x0000000106210084(Flutter + 0x004dc084)
0x000000010620fc68(Flutter + 0x004dbc68)
0x0000000106291920(Flutter + 0x0055d920)
0x000000010624a014(Flutter + 0x00516014)
0x00000001c1ff52bc(libsystem_pthread.dylib + 0x0000b2bc)_pthread_body
0x00000001c1ff521c(libsystem_pthread.dylib + 0x0000b21c)_pthread_start
```

Symbolicating here.

**Comment by mehmetf** [comment]
@flutter-symbolizer-bot https://github.com/flutter/flutter/issues/69683#issuecomment-729869926 internal engine#073263e39d1d12313c9622ac3788cc9f2b01f582 ios release arm64

**Comment by mehmetf** [comment]
Yep, as suspected, same thing.

**Comment by mehmetf** [comment]
Fix for this landed with cl 340967775.
